### PR TITLE
update readme for apps/fraud-detection

### PR DESCRIPTION
**Pull request by yunyanliluo**

### CODE REVIEW

File: apps/fraud-detection/README.md [hunk 1]
```diff
@@ -23,13 +23,18 @@ jupyter toree install --spark_opts='--master=local[*]' --user --spark_home=$SPAR
...
 
-4. Run the `$ANALYTICS_ZOO_HOME/apps/fraud-detection/run.sh` to start the jupyter notebook. Change parameter settings as you need, ie `MASTER = local[physcial_core_number]`.
-Recommended driver memory and executor memory is 10g. 
-
+```bash
+MASTER=...
+${ANALYTICS_ZOO_HOME}/bin/spark-shell-with-zoo.sh \
+    --master ${MASTER} \
+    --driver-memory 10g \
+    --executor-memory 10g \
```
Comment (hhbyyh) [inline apps/fraud-detection/README.md]: I don't think the script here can start the notebook.

File: apps/fraud-detection/README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ how to use scala in Jupyter notebook.
 
 An outline is:
 ```bash
-pip install https://dist.apache.org/repos/dist/dev/incubator/toree/0.2.0-incubating-rc6/toree-pip/toree-0.2.0.tar.gz
+pip install https://dist.apache.org/repos/dist/dev/incubator/toree/0.3.0-incubating-rc1/toree-pip/toree-0.3.0.tar.gz
```
Comment (hhbyyh) [inline apps/fraud-detection/README.md]: Hi @yunyanliluo just want to check if you have tested it?

Comment (yunyanliluo) [reply apps/fraud-detection/README.md]: I have successfully installed toree-0.3.0 locally. The original link of toree-0.2.0 has failed.

Comment (hhbyyh) [reply apps/fraud-detection/README.md]: OK.

**Comment by hhbyyh** [comment]
LGTM

**Comment by hkvision** [comment]
Please backport to branch-0.3 @yunyanliluo

**Pull request merged**